### PR TITLE
update Predict modality

### PR DESCRIPTION
**Pull request by KaiWaldrant**
## Describe your changes
update predict modality nextflow

## Issue ticket number and link
Closes #xxxx (Replace xxxx with the GitHub issue number)

## Checklist before requesting a review
- [ ] I have performed a self-review of my code

- Check the correct box. Does this PR contain:
  - [ ] Breaking changes
  - [ ] New functionality
  - [ ] Major changes
  - [ ] Minor changes
  - [ ] Bug fixes

- [ ] Proposed changes are described in the CHANGELOG.md

- [ ] CI Tests succeed and look good!

### CODE REVIEW

File: nxf-tmp.5mFiyd
Comment (rcannood) [inline nxf-tmp.5mFiyd:1]: todo: remove

File: src/datasets/resource_test_scripts/bmmc_x_starter.sh [hunk 1]
```diff
@@ -1,20 +1,33 @@
 #!/bin/bash
 
-NEURIPS2021_URL="https://github.com/openproblems-bio/neurips2021_multimodal_viash/raw/main/resources_test/common"
-DATASET_DIR="resources_test/common"
+# NEURIPS2021_URL="https://github.com/openproblems-bio/neurips2021_multimodal_viash/raw/main/resources_test/common"
```
Comment (rcannood) [inline src/datasets/resource_test_scripts/bmmc_x_starter.sh:3]: did you mean to uncomment this?

File: src/tasks/denoising/workflows/run_benchmark/main.nf [hunk 2]
```diff
@@ -93,7 +93,7 @@ workflow run_wf {
       def new_id = "output"
       def new_state = [
         input: states.collect{it.metric_output},
-        output: states[0].output
+        output: states[0]._meta
```
Comment (rcannood) [inline src/tasks/denoising/workflows/run_benchmark/main.nf:96]:
```suggestion
        _meta: states[0]._meta
```

File: src/tasks/predict_modality/resources_test_scripts/bmmc_x_starter.sh [hunk 3]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
-#
-#make sure the following command has been executed
-#viash ns build -q 'predict_modality|common' --parallel --setup cb
+
+# Run this prior to executing this script:
+# bin/viash_build -q 'batch_integration'
```
Comment (rcannood) [inline src/tasks/predict_modality/resources_test_scripts/bmmc_x_starter.sh:4]: Remove outdated docs

```suggestion
```

**Review by rcannood — CHANGES_REQUESTED** [review]
minor changes requested

File: src/datasets/resource_test_scripts/bmmc_x_starter.sh (hunk 1 above)
Comment (KaiWaldrant) [reply src/datasets/resource_test_scripts/bmmc_x_starter.sh:3]: yes

**Review by rcannood — APPROVED** [review]

**Pull request merged**